Repo structure:
    This repo contains three main elements:
    1.	Data directory (“data”) which contains all input data for computational analysis and a readme file that contains more detailed information about each data file.
    2.	Source code directory (“code”) which contains all necessary code for generating differential expressed gene list and generating three main plots in our study. A readme file is also included to specify the purpose of each code file.
    3.	Readme file which contains information of the overall narrative of this project and explanation of the computational pipeline that leads to generation of the DEG lists and figures


Guide for recreating computational results:
   1.Ranked DEG Comparison Test
    The ranked DEG comparison test reflects whether or not the overlapping genes between male and female DEGs are more highly differentially expressed compared to the non-overlapping genes. A plot was generated by first ranking the DEG dataset by p-value and compute the number of overlaps between the top 50 DEGs of male and female, then repeated for the top 100, 200… up to 900 genes.
    A.	Start by downloading Female_published_DEG_list and MaleDEG_published_DEG_list in the data folder
    B.	Run DEG_Rank_Test.ipynb in the code folder
   
   2.Differentially Expressed Gene List generation and comparison between male and female
    A.	Download GSE102556_HumanMDD_fpkmtab.csv from the data folder
    B.	Run Limma_deg_resampling.Rmd
    Limma_deg_script_NAc.Rmd
    Limma_deg_script_antins.Rmd
    Limma_deg_script_ba11.Rmd
    Limma_deg_script_ba25.Rmd
    Limma_deg_script_ba89_dlPFC.Rmd
    Limma_deg_script_subic.Rmd
    C.	From the last step, six separate lists of DEG will be generated from all six brain regions included in the original dataset and for “antins”, resampling method will be applied for 1000 permutations to generate variations of DEG identification within the same sex, at the same brain region
    D.	Download DEG comparision OG.ipynb
    E.	Move all output files from step B to the same directory as the ipynb file from step D
    F.	Run DEG comparision OG.ipynb
    G.	Collect results of Male/Female DEG overlaps and probability density plots for the resampling step. 

   3.Multidimensional scaling (MDS) on new whole brain MDD gene expression profiling dataset
    A.	Download GSE101521_totalRNA_counts.cvs.gz from the data folder, decompress once downloaded
    B.	Run MDS_limma_seconddata.rmd, collect MDS plot from the pipeline
    C.	Move .csv result file to the same directory as DEG comparison OG.ipynb
    D.	Generate DEG comparison for the new dataset by running DEG comparison OG.ipynb 



Citation for project overview and computational methods:
   A. Big picture inspiration for examining statistical power of sexual dimorphism related claims in recent studies:
    Clayton, J. A., & Collins, F. S. (2014). Policy: NIH to balance sex in cell and animal studies. Nature, 509(7500), 282-283. doi:10.1038/509282a
    https://www.nature.com/news/policy-nih-to-balance-sex-in-cell-and-animal-studies-1.15195 

   B. Datasets: 
    Main dataset which includes bulk RNA seq data from a cohort of 46 patients, pormartal brain tissues specimen from public brain banks: 
    Labonte et al (2017), Sex-specific transcriptional signatures in human depression, Nat Medicine 2017 https://www.nature.com/articles/nm.4386#methods 
    Second dataset for DEG overlap validation from a separate study which includes MDD brain bulk RNA seq data in additional to RNA seq data from non-suidcide control group: 
    Pantazotos et al (2017), Whole-transcriptome brain expression and exon-usage profiling in major depression and suicide, Mol Psychiatry
    https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313378/

   C. Computational methods: 
    RNA-Seq differential expression analysis: An extended review and a software tool, Plos One 2017
    https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0190152 
    Smyth, G. K., et al (2019).  Limma: Linear Models for Microarray and RNA-Seq Data User’s Guide
    https://www.bioconductor.org/packages/devel/bioc/vignettes/limma/inst/doc/usersguide.pdf
    Law, C. et al (2018). RNA-seq analysis is easy as 1-2-3 with limma, Glimma and edgeR. .https://www.bioconductor.org/packages/devel/workflows/vignettes/RNAseq123/inst/doc/limmaWorkflow.html?fbclid=IwAR2WVCMVdGdE203rAldtpxo8g043Li1toaiS4TnHU-Fo0QtxEQUvYjEA9KM
    A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis, Briefings in Bioinformatics 2013.  https://academic.oup.com/bib/article/14/6/671/189645
